NOTICE

```diff
@@ -4,5 +4,6 @@ Copyright 2015 - 2016 The Apache Software Foundation
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
 
+Portions of this software were developed at Google (http://www.google.com/). Licensed under the Apache v2.0 License.
```

Comment:
I think we better be more specific here and add pah `./bigquery/` to avoid the confusion.